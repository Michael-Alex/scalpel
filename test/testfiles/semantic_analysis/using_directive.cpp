namespace n
{
	class c{};
}

using namespace n;

c a_c;

