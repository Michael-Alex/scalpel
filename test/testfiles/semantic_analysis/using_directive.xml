<namespace id="n0">
 <namespace name="n" id="n1">
  <class name="c" id="c0">
   <destructor access="public" defined="false">
   </destructor>
  </class>
 </namespace>
 <variable name="a_c">
  <type>
   <class id="c0"/>
  </type>
 </variable>
</namespace>
