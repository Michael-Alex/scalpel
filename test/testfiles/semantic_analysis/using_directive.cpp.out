<namespace id="0">
 <namespace name="n" id="1">
  <class name="c" id="0">
   <destructor access="public" defined="false">
   </destructor>
  </class>
 </namespace>
 <variable name="a_c">
  <type>
   <class id="0"/>
  </type>
 </variable>
</namespace>
